package.json

```diff
@@ -195,6 +195,7 @@
     "makelogs": "3.0.2",
     "marked-text-renderer": "0.1.0",
     "mocha": "2.5.3",
+    "murmurhash3js": "3.0.1",
```

Comment:
What are we using for the hashes in the upcoming state PR?  Can we use the same dependency?